.github/workflows/build_strelka_on_push.yml

```diff
@@ -1,6 +1,9 @@
 name: "Push Build"
```

Comment:
nit: this is confusing to me cause it sounds like it's pushing a build, but it's just building the images. same problem for the job name

Reply:
Didn't change here/wasn't named by us. I agree it's confusing, if you want to change it go ahead.